source/_integrations/weather.markdown

```diff
@@ -34,3 +34,9 @@ The `weather` platform only knows the below listed conditions. The reason for th
 - 'windy'
 - 'windy-variant'
 - 'exceptional'
+
+### Forecast Weather Conditions Intervals
+
+Periodic forecast information is stored in the `forecast` attribute on the entity. To access and use the information should be reserved for advanced users using [Templates](/docs/configuration/templating/).
+
+![Example OpenWeatherMap forecast attributes](/images/integrations/openweathermap/OpenWeatherMap_Attributes.png)
```

Comment:
Can we inline a bit of a state YAML as an example instead? Screenshots are easy to get outdated.

Reply:
Good @frenck I will do that.

Reply:
Replaced image with YAML example. 3e25e8a57560e47829d5779f21f9806477a3674a